converters/fromZigbee.js

```diff
@@ -698,6 +698,44 @@ const converters = {
             }
         },
     },
+    ZNCLDJ11LM_curtain_genAnalogOutput_change: {
+        cid: 'genAnalogOutput',
+        type: 'attReport',
+        convert: (model, msg, publish, options) => {
+            let running = false;
+            if (msg.data.data['61440']) {
+                if ((msg.data.data['61440'].toString()).startsWith('117440')
+            ||(msg.data.data['61440'].toString()).startsWith('591')) {
+                    running = true;
+                } else if (msg.data.data['61440']===0) {
+                    running = false;
+                }
+
+                return {presentValue: precisionRound(msg.data.data['presentValue'], 2), running: running};
+            }
+            return {presentValue: precisionRound(msg.data.data['presentValue'], 2),
+                running: running};
+        },
+    },
+    ZNCLDJ11LM_curtain_closuresWindowCovering_change: {
+        cid: 'closuresWindowCovering',
+        type: 'attReport',
+        convert: (model, msg, publish, options) => {
+            if (msg.data.data['currentPositionLiftPercentage']) {
+                const lift = precisionRound(msg.data.data['currentPositionLiftPercentage'], 2);
+                return {currentPositionLiftPercentage: lift};
```

Comment:
Whats does this mean, also could you change this to lift:

Reply:
The currentPositionLiftPercentage does not change correctly. I therefore added closuresWindowCovering attReport to ignore for now.